cmake/toolchain-mingw-i686.cmake

```diff
@@ -24,14 +24,14 @@ set(CMAKE_C_COMPILER ${C_COMPILER_FULL_PATH})
 
 find_program(CXX_COMPILER_FULL_PATH NAMES
     g++-mingw-w64-${CMAKE_CXX_COMPILER_TARGET}
```

Comment:
This like needs also be corrected...

Reply:
I'm not aware of any distro that uses such a naming scheme, so I'll remove it.